e2e/utils/pages/homepage.page.ts

```diff
@@ -228,25 +228,23 @@ abstract class Homepage {
 		if (await this.isVisibleByTestId(testId)) {
 			await this.clickByTestId(testId);
 		} else {
-			if (await this.isVisibleByTestId(NAVIGATION_MENU_BUTTON)) {
-				await this.clickByTestId(NAVIGATION_MENU_BUTTON);
-			}
-			if (await this.isVisibleByTestId(testId)) {
-				await this.clickByTestId(testId);
-			}
+			const navigationMenuButton = this.#page.getByTestId(NAVIGATION_MENU_BUTTON);
+			await navigationMenuButton.click();
+			const navigationMenu = this.#page.getByTestId(NAVIGATION_MENU);
+			await navigationMenu.getByTestId(testId).click();
 		}
 	}
 
 	async activateTestnetSettings(): Promise<void> {
 		await this.navigateTo(NAVIGATION_ITEM_SETTINGS);
 		await this.clickByTestId(BTC_TESTNET_TOGGLE);
-		await this.clickByTestId(NAVIGATION_ITEM_TOKENS);
+		await this.clickByTestId(NAVIGATION_ITEM_HOMEPAGE);
 	}
 
-	async takeScreenshot(): Promise<void> {
+	async takeScreenshot(isMobile: boolean): Promise<void> {
 		await expect(this.#page).toHaveScreenshot({
 			// creates a snapshot as a fullPage and not just certain parts.
-			fullPage: true,
+			fullPage: !isMobile,
```

Comment:
Would be great to capture full pages for mobile too. Any particular reason you disabled it?

Reply:
Yeah, guessed so, but I think we still need to have keep it as `true` - on mobile, most of the content will be hidden if we don't do so, therefore we would be ignoring 60%+ of a page without a full snapshot.